util/ckan_loader.py

```diff
@@ -0,0 +1,46 @@
+import json
+import logging
+import os
+
+import ckanapi
+
+DEMO_DATA_PATH = os.path.join(os.path.dirname(os.path.abspath(__file__)), '..')
```

Comment:
Just so I understand correctly.  Why are you saying the demo data path should be the parent directory of the "demo_data" folder?

Reply:
You're absolutely right. My mistake. Thanks.